packages/suite/src/views/suite/bridge-deprecated/index.tsx

```
@@ -0,0 +1,50 @@
+            isHeadingCentered
+            bottomBarComponents={
+                <>
+                    <Button
+                        icon="caretLeft"
+                        variant="tertiary"
+                        onClick={() => goToWallet()}
+                        data-testid="@bridge/goto/wallet-index"
+                    >
+                        <Translation id="TR_TAKE_ME_BACK_TO_WALLET" />
+                    </Button>
+                </>
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)